src/components/Autocomplete/README.md

```diff
@@ -301,6 +302,128 @@ class AutocompleteExample extends React.Component {
 }
 ```
 
+### Autocomplete with lazy loading
+
+```jsx
+class AutoCompleteLazyLoadExample extends React.Component {
+  paginationInterval = 25;
+
+  options = Array.from(Array(1000)).map((_, index) => ({
```

Comment:
Should we reduce `1000` to `500` or `300`? After loading 500 the list is extremely janky

Reply:
Yeah, I noticed while playing around with this example in the deployment yesterday that with large lists (and maybe regardless of list length) there's a bug with selection not working onClick and instead scrolling back to the top of the list that again doesn't seem to be an issue with keyboard, but that's another problem for another PR...